src/radar_control.py

```diff
@@ -358,7 +357,6 @@ def search_for_experiment(radar_control_to_exp_handler, exphan_to_radctrl_iden,
             radar_control_to_exp_handler,
             exphan_to_radctrl_iden,
             log,
-            expected_type=ExperimentPrototype,
```

Comment:
Why was this removed?

Reply:
The experiment returned is either of the `ExperimentPrototype`  class, or is type `None`. So when there is no new experiment it returns `None` and then throws and error because that is the wrong type. For now I've put a TODO in `socket_operations.recv_pyobj` to be able to handle defining multiple acceptable types .

Reply:
Ahhh that makes sense. Thanks for explaining